### DISCUSSION TITLE
Cannot use firebase storage in with wordpress

### DISCUSSION THREAD

**Discussion by hengseyha** [dalenguyen/firebase-wordpress-plugin#182] [General]
I try to upload image to firebase storage but function not a function

**Comment by dalenguyen** [comment]
Hi @hengseyha, can you send a snapshot of the error & the version of the plugin that you're using?

**Reply by hengseyha** [reply]
Now it is working, I add storage js to plugin then the service working fine. thank you

**Reply by dalenguyen** [reply]
Awesome. I'm glad that it works. There is option in the General tab. I guess that you forget to check it. 

![image](https://user-images.githubusercontent.com/14116156/139269448-883eda28-fb34-4c73-85df-4f53553b9c6f.png)

**Reply by hengseyha** [reply]
OH, sorry I missed this section

**Reply by hengseyha** [reply]
cold I ask more about add data to firebase? I want after post success, I use Ali to last update json data in order to set to firebase but I don't which hook function is to be used. I use add_action('post_save') not get last save data.

**Reply by dalenguyen** [reply]
Here is the hook that helps to save data to firebase. 

https://firebase-wordpress-docs.readthedocs.io/en/latest/hooks/pro-filters.html#save-data-to-firebase
